README.md

```diff
@@ -1,25 +1,38 @@
+[![SublimeHQ Discord](https://img.shields.io/discord/280102180189634562?label=SublimeHQ%20Discord&logo=discord&style=flat-square)](https://discord.gg/D43Pecu)
+
 # Sublime Packages
 
-> **These packages are developed against the latest [Sublime Text 3 Dev Build](http://sublimetext.com/3dev). Bugs may exist on older builds, and the format used is not compatible with builds older than 3092.**
+> **These packages are developed against the latest [Sublime Text Dev Build](http://sublimetext.com/).**
```

Comment:
This should say:

```suggestion
> **These packages are developed against the latest dev build of Sublime Text, currently build 4094. These syntaxes may be incompatible, sometimes in very subtle ways, with older builds.**
```

We'll leave the link out for now since we don't want to publicly link to downloads.